File: app/src/main/java/com/example/demo2/Web/net/revert/BaseResponseEntity.java
package com.example.demo2.Web.net.revert;

import com.example.demo2.Web.utils.ApiConfig;

import java.io.Serializable;

public class BaseResponseEntity implements Serializable {

  private static final long serialVersionUID = 1L;

  public int code;

  public String msg;

  public static long getSerialVersionUID() {
    return serialVersionUID;
  }

  public int getCode() {
    return code;
  }

  public void setCode(int code) {
    this.code = code;
  }

  public String getMsg() {
    return msg;
  }

  public void setMsg(String msg) {
    this.msg = msg;
  }

  public boolean success() {
    return ApiConfig.getSucceedCode() == code;
  }

  public boolean tokenInvalid() {
    return ApiConfig.getInvalidateToken() == code;
  }

}

File: app/src/main/java/com/example/demo2/Web/net/revert/BaseResponseEntity.kt
//package com.mp5a5.www.library.net.revert
//
//import com.mp5a5.www.library.utils.ApiConfig
//import java.io.Serializable
//
///**
// * @author ：mp5a5 on 2019/4/16 19：00
// * @describe
// * @email：wwb199055@126.com
// */
//open class BaseResponseEntity : Serializable {
//
//    open var code: Int = -1
//
//    open var msg: String = ""
//
//    companion object {
//        private const val serialVersionUID = 1L
//    }
//
//    open fun success(): Boolean {
//        return ApiConfig.getSucceedCode() == code
//    }
//
//    open fun tokenInvalid(): Boolean {
//        return ApiConfig.getInvalidateToken() == code
//    }
//
//}
